arctic/arctic.py

```diff
@@ -202,6 +207,44 @@ def list_libraries(self):
                         libs.append(coll[:-1 * len(self.METADATA_COLL) - 1])
         return libs
 
+    @mongo_retry
+    def invalidate_cache(self):
+        """
+        Deletes the data in the collection used to cache the result
+        of list_libraries
+        :return:
+        """
+        self._conn.meta_db.cache.drop()
```

Comment:
Let's discuss tomorrow, I think have the cache exist over time is still preferable and simpler than deleting dbs or collections.